shared/desktop/renderer/remote-component-loader.js

```diff
@@ -17,9 +17,9 @@ import {globalColors} from '../../styles'
 import {initAvatarLookup, initAvatarLoad} from '../../common-adapters'
 import {remote, ipcRenderer} from 'electron'
 import {setupContextMenu} from '../app/menu-helper'
-import {setupSource} from '../../util/forward-logs'
 
-setupSource()
+// TODO add a logger for the remote components
```

Comment:
these TODOs can likely go away as the new model is the remotes (launcher included) are 'dumb' now and everything goes through the main window (and its logger)

Reply:
Sweet